components/result/result.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+
+    <link rel="stylesheet" href="https://rapidui.netlify.app/styles/main.css" />
+
+    <link rel="stylesheet" href="../../../index.css" />
+    <link rel="stylesheet" href="../quizzes/quizzes.css" />
+    <link rel="stylesheet" href="./result.css" />
+
+    <title>Rapid Quiz</title>
+  </head>
+  <body>
+    <div class="page-layout">
+      <nav class="quiz-navbar">
+        <div class="logo">
+          <a href="#" class="quiz-navbar-text">
+            <span class="font-family-kaushan">R</span>apid
+            <span class="font-family-kaushan">Q</span>uiz</a
+          >
+        </div>
+      </nav>
+
+      <!-- --- main body ---- -->
+      <main class="result-container">
+        <h1 class="justify-center mb-1">Result</h1>
+        <img
+          src="../../assets/img/trophy.png"
```

Comment:
you can use the `loading=lazy` attribute with `<img />`. It will help in improving performance.

Reply:
Oh yes, I missed it there !